### ISSUE TITLE
Don't recreate output directory if it already exists

### ISSUE THREAD

**Issue by toothstone** [mumax/3#281]
TL;DR: I would like to see mumax3 keeping an existing output directory intact, especially its UNIX file permissions.
If somebody would be so nice as to point me in the right direction on the codebase, I could look into creating a PR myself.

I'm currently working on a wrapper application for simulation suites, and from what I've seen so far mumax3 recreates the `.out` output directory when it is called, no matter what is already present.

This doesn't play nice with how I handle permissions, and I would suspect it is generally disruptive, as the directory file handle becomes stale on restarting mumax, and all files in the output directory are lost, even if they are not related to mumax3.

**Comment by godsic** [comment]
@toothstone if `-f` flags is not provided, then `mumax3` should not re-create the output directory. Does it work for you?

**Comment by toothstone** [comment]
Setting `-f=False` seems to do the trick in general, just not in my specific use case, I suspect some hidden interactions between the non-interactive shell used and/or the umask I have to set.
But thanks for the hint, I'll close this issue for now - although I think this should not be the default behaviour, but I guess making a breaking change at this point is not a good idea, either.
